pseudomonas_analysis/nbconverted/2_identify_generic_genes_pathways.py

```diff
@@ -304,6 +311,132 @@
 # 2. An enrichment score (ES) is defined as the maximum distance from the middle of the ranked list. Thus, the enrichment score indicates whether the genes contained in a gene set are clustered towards the beginning or the end of the ranked list (indicating a correlation with change in expression). 
 # 3. Estimate the statistical significance of the ES by a phenotypic-based permutation test in order to produce a null distribution for the ES( i.e. scores based on permuted phenotype)
 
+# In[21]:
+
+
+# Load pathway data
+adage_kegg_DB_filename = params['pathway_DB_filename']
+
+
+# In[22]:
+
+
+# Need to format data into tab-delimited matrix
+# with columns= KEGG pathway name, description, gene ids
+# Each gene ids is tab separated
+adage_kegg_DB_processed_filename = os.path.join(
+    base_dir,
+    dataset_name,
+    "data",
+    "metadata",
+    "adage_kegg_DB_process_filename.gmt"
+)
+process.format_pseudomonas_pathway_DB(adage_kegg_DB_filename, local_dir, adage_kegg_DB_processed_filename)
+
+
+# In[23]:
+
+
+get_ipython().run_cell_magic('R', '-i base_dir -i template_DE_stats_filename -i adage_kegg_DB_processed_filename -i statistic -o template_enriched_pathways', "\nsource(paste0(base_dir, '/generic_expression_patterns_modules/GSEA_analysis.R'))\ntemplate_enriched_pathways <- find_enriched_pathways(template_DE_stats_filename, adage_kegg_DB_processed_filename, statistic)")
+
+
+# In[24]:
+
+
+print(template_enriched_pathways.shape)
+template_enriched_pathways[template_enriched_pathways['padj'] < 0.05].sort_values(by='padj')
```

Comment:
Do you want to print the whole table here, or would the first few rows (`df.head()`) be enough?